source/_integrations/vitrea.markdown

```diff
@@ -0,0 +1,110 @@
+---
+title: Vitrea
+description: Instructions on how to integrate Vitrea home automation devices with Home Assistant.
+ha_category:
+  - Cover
+  - Switch
+  - Number
+ha_release: 2025.9
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - "@solangek"
+ha_domain: vitrea
+ha_platforms:
+  - cover
+  - number
+  - switch
+ha_integration_type: hub
+---
+
+The Vitrea integration allows you to control Vitrea home automation devices through Home Assistant. This integration supports switches, covers (blinds), and timer (boiler) controls for your Vitrea system.
```

Comment:
```suggestion
The **Vitrea** {% term integration %} allows you to control Vitrea home automation devices through Home Assistant. This integration supports switches, covers (blinds), and timer (boiler) controls for your Vitrea system.
```
There's a [glossary term reference](https://developers.home-assistant.io/docs/documenting/integration-docs-examples#glossary-term-reference) for the term "integration".